### PR TITLE
Add a bit of documentation for d2o and d3o commands.

### PR DESCRIPTION
**Pull request by mcepl**
Covering changes in #801.

### CODE REVIEW

**Comment by justinmk** [comment]
The documentation should probably live under `:Gblame` next to the other mappings. https://github.com/tpope/vim-fugitive/pull/854/files#diff-2d13dec36bf619dd45b9bf4d1bd631a5R208

**Comment by mcepl** [comment]
> The documentation should probably live under :Gblame next to the other mappings.

I don’t understand. ``Gdiff`` mapping already has documentation above those mappings in the same section of the file. We probably may need to switch this documentation in a separate section if anything, but I don’t understand how moving it couple of lines down helps anything.

**Comment by justinmk** [comment]
Ok, but at least they should not be surround in single-quotes (`'d2o'`), that's reserved for options--not commands or mappings--in Vim docs.

**Comment by mcepl** [comment]
Better?

**Comment by tpope** [comment]
Can you make it one big paragraph please?

File: doc/fugitive.txt [hunk 1]
```diff
@@ -182,6 +182,9 @@ that are part of Git repositories).
                         the width of the window relative to 'textwidth'.  Use
                         |do| and |dp| and write to the index file to simulate
                         "git add --patch".
+                        For the three-way diff, there is also |d2o| and |d3o|
```
Comment (justinmk) [inline doc/fugitive.txt]: Bars are for tags, these tags go to nowhere. Perhaps backticks:

    `d2o`

or double quotes:

    "d2o"

Comment (mcepl) [inline doc/fugitive.txt]: How is it different from |do| and |dp| just a line above?

**Comment by mcepl** [comment]
OK, I think this is the proper way how to do it.

**Comment by mcepl** [comment]
BTW, ping?

**Comment by tpope** [comment]
Remind me, how did we get from "Can you make it one big paragraph" to a subsection in a completely different part of the file?

**Comment by mcepl** [comment]
I am afraid I lost that during those five months you missed this PR ;).

Sorry, I probably really did not understand what you expected. Could you repeat, please, or elaborate what did you expect to be in that paragraph? Should I rewrite ``:diffget`` help in the form appropriate for ``d2o``?

**Comment by tpope** [comment]
Piecing things together from my emails, the patch originally looked something like
```
@@ -182,6 +182,9 @@ that are part of Git repositories).
                         the width of the window relative to 'textwidth'.  Use
                         |do| and |dp| and write to the index file to simulate
                         "git add --patch".
+                        For the three-way diff, there is also |d2o| and |d3o|
```

All I wanted was for you to take `For the next...` and put it on the same line as `"git add --patch".`. Also @justinmk told you to drop the `|`s because those aren't valid help tags, and that still stands.

**Comment by mcepl** [comment]
> All I wanted was for you to take For the next... and put it on the same line as "git add --patch"..

Aaaaahhh! I see. OK, I can certainly do it.

> Also @justinmk told you to drop the |s because those aren't valid help tags, and that still stands.

And why is it already OK in the same paragraph for ``do`` and ``dp`` commands? What’s the difference?

**Comment by tpope** [comment]
Compare `:help do` and `:help dp` to `:help d2o` and `:help d3o`.

**Comment by tpope** [comment]
Oh, and in accordance with the contribution guidelines, drop the period in your commit subject.

**Comment by mcepl** [comment]
What about this one?

**Comment by tpope** [comment]
How many times are you going to rip the nice simple singular paragraph away from me

**Comment by mcepl** [comment]
Whatever you say, sir!

**Comment by mcepl** [comment]
Thank you and sorry for being such a bother.

**Pull request merged**